src/scene_battle_rpg2k3.cpp

```diff
@@ -976,7 +976,7 @@ bool Scene_Battle_Rpg2k3::CheckWin() {
 		std::vector<int> drops;
 		Main_Data::game_enemyparty->GenerateDrops(drops);
 
-		Game_Message::texts.push_back(Data::terms.victory);
+		Game_Message::texts.push_back(Data::terms.victory + "\\|");
```

Comment:
I think this will need Player::escape_char instead of "\\" because when japanese encoding is used the escape char is the Yen sign :/
(Testing this later)